packages/atomic/src/locales.json

```diff
@@ -700,32 +700,32 @@
     "fr": "Segments d'URI réduits"
   },
   "disconnected": {
-    "en": "Could not connect.",
-    "fr": "Impossible de se connecter."
+    "en": "No Internet access.",
+    "fr": "Pas d'accès internet."
   },
   "check-your-connection": {
     "en": "Your query couldn't be sent. Verify your internet connection.",
     "fr": "Votre requête n'a pas plus être envoyée. Peut-être que vous avez perdu votre connexion internet."
   },
   "no-endpoints": {
-    "en": "The Coveo Organization has no registered endpoints.",
-    "fr": "L'organisation Coveo n'a pas de point de terminaison unique."
+    "en": "Your organization {{org}} has no available content.",
+    "fr": "Votre organisation {{org}} n'a pas contenu disponible."
```

Comment:
Should the key still be `no-endpoints` now that the message is about missing content?

Reply:
No endpoints means you have nothing to return / nothing indexed, I think.

I don't want to change the key because it could be seen as a breaking change. Description also provides more details:

```
 "add-sources": {
    "en": "Add content sources or wait for your newly created sources to finish indexing.",
    "fr": "Vous devrez ajouter des sources à votre index, ou attendre la fin de l'indexation des sources créées."
  },
```